libkbfs/tlf_journal.go

```diff
@@ -560,6 +611,20 @@ func (j *tlfJournal) resumeBackgroundWork() {
 	}
 }
 
+func (j *tlfJournal) resumeBackgroundWork() {
```

Comment:
Analogously to `pauseBackgroundWork(pauseType)`, I think we can just have a single function

```
resumeBackgroundWork(pauseType) {
  j.pausedType &= ^pauseType
  ...
  select {
    ... j.needResumeCh <- struct{}{}
  }
}
```